docs/source/killed.rst

```diff
@@ -0,0 +1,105 @@
+other workers, if there are any left.
+
+You should expect to see the following message at the end of the worker's log:
+
+.. pull-quote::
+
+   distributed.dask_worker - INFO - End worker
+
+In these cases, there is not normally anything which you need to do, since the
+behaviour is expected.
+
+Unrecoverable Exception
+'''''''''''''''''''''''
+
+The worker is a python process, and like any other code, an exception may occur
+which causes the process to exit. One typical example of this might be a
+version mismatch between the packages of the client and worker, so that
+a message sent to the worker errors while being unpacked. There are a number of
+packages that need to match, not only ``dask`` and ``distributed``.
+
+In this case, you should expect to see the full python traceback in the worker's
+log. In the event of a version mismatch, this might be complaining about a bad
+import or missing attribute. However, other fatal exceptions are also possible,
+such as trying to allocate more memory than the system has available, or writing
+temporary files without appropriate permissions.
+
+To assure that you have matching versions, you should run (more recent versions
+of distributed may do this automatically)
+
+.. code-block::
+
+   client.get_versions(check=True)
+
+For other errors, you might want to run the computation in your local client, if
+possible, or try grabbing just the task that errored and using
+:func:`recreate_error_locally <distributed.recreate_exceptions.ReplayExceptionClient.recreate_error_locally>`,
+as you would for ordinary exceptions happening during task execution.
+
+Specifically for connectivity problems (e.g., timeout exceptions in the worker
+logs), you will need to diagnose your networking infrastructure, which is more
+complicated than can be described here. Commonly, it may involve logging into
+the machine running the affected worker
+(although you can :ref:`ipylaunch`).
+
+Sudden Exit
+'''''''''''
+
+The worker process may stop working without notice. This can happen due to
+something internal to the worker, e.g., a memory violation (common if interfacing
+with compiled code), or due to something external, e.g., the ``kill`` command, or
+stopping of the container or machine on which the worker is running.
+
+In the best case, you may have a line in the logs from the OS saying that the
+worker was shut down, such as the single word "killed"  or something more descriptive.
+In these cases, the fault may well be in your code, and you might be able to use the
+same debugging tools as in the previous section.
+
+However, if the action was initiated by some outside framework, then the worker will
+have no time to leave a logging message, and the death *may* have nothing to do with
+what the worker was doing at the time. For example, if kubernetes decides to evict a
+pod, or your ec2 instance goes down for maintenance, the worker is not at fault.
```

Comment:
I think that these typically leave a kill signal that often has as much or more information than a segfault